engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -572,7 +572,9 @@ private void terminateElement(
     } else if (elementType == BpmnElementType.CALL_ACTIVITY) {
       final var calledActivityElementInstance =
           elementInstanceState.getInstance(elementInstance.getCalledChildInstanceKey());
-      terminateElement(calledActivityElementInstance, sideEffects);
+      if (calledActivityElementInstance != null) {
```

Comment:
:wrench: We could apply the same check `.canTerminate()` as for the other child instances.

Reply:
Yes, we could add `canTerminate()` additionally to the null check. :sweat_smile: